doc/man/lttng-enable-event.1.txt

```diff
@@ -410,8 +410,10 @@ Dynamic user space probes
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 With the option:--userspace-probe option, you can instrument function
 entries of any user space binary (application or library) using either
-an available symbol name or a SystemTap SDT probe's provider and probe
-names.
+an available symbol name or a SystemTap Statically Defined Tracepoints
+(SDT, a DTrace-style marker) probe's provider and probe names. As of
+this version, only SDT probes that are :not: surrounded by a reference
+counter (semaphore).
```

Comment:
Is there a "are supported" missing here?

Reply:
Fixed in 9a49aca396e28c579e2fd143693edb917f39631b.

Reply:
Fixed in 22337deae7794dcd623888e9beee89d886272165.